src/Exceptions.py

```diff
@@ -152,4 +152,19 @@ def __init__(self, aFun, someArgs, lineNo=-1, aCls=None):
     def __str__(self):
         ret = super().__str__()
         ret += "\n\tWith args: " + str(self.argLst)
+        return ret
+    
+class NonIntIndexException(PyDuckerError):
+    
+    def __init__(self, someName, lineNo=-1):
+        """
+        @someName:str
+        @lineNo:int
+        """
+        super().__init__(lineNo)
+        self.aName = someName
+        
+    def __str__(self):
+        ret = super().__str__()
+        ret += "\n\tVariable name: " + self.aName + " is not an integer and needs to be to index."
```

Comment:
Would it also be helpful to include the type of the thing that was indexing here.

Would need to change the __init__ to have someName be a varBean, then only need to access the internals to accomplish this.